x/campaign/keeper/campaign.go

```diff
@@ -70,6 +70,12 @@ func (k Keeper) GetCampaign(ctx sdk.Context, id uint64) (val types.Campaign, fou
 	return val, true
 }
 
+// RemoveCampaign removes a campaign from the store
+func (k Keeper) RemoveCampaign(ctx sdk.Context, id uint64) {
```

Comment:
Is this function only being used internally for tests?

Reply:
Yes, we could actually have it in the test package as a utility but I would think it's fine this way since `campaign.go` contains general low level methods to interact with campaigns in the store